man/osg-incommon-cert-request.1

```diff
@@ -44,41 +46,63 @@ May be specified more than once for additional SANs.
 .TP
 .BR \-d, \-\-directory
 The directory to write the host certificate(s) and key(s).
+.TP
+.BR \-O, \-\-orgcode
+Specify alternative organization and department codes for the InCommon \\
+Certificate Service. Defaults are Fermilab\'s codes.
 .SH EXAMPLES
 .PP
 To generate a certificate with no Subject Alternative Names (SANS)
 .PP
 .RS
 .nf
-osg-incommon-cert-request -u incommonuser -c certpath.pem -k keypath.pem -H hostname.yourdomain
+osg-incommon-cert-request -u incommonuser -c certpath.pem -k keypath.pem \\
+-H hostname.yourdomain
```

Comment:
Indent this line 4 spaces.  The `.nf` preserves the spaces.

Reply:
You missed the indent on continued lines.

Reply:
Please check if the indents are correct.